bentoml/_internal/frameworks/tensorflow_v2.py

```diff
@@ -304,23 +286,25 @@ def plus_one(x):
         return _model.tag
 
 
-def save(
+def save_model(
     name: str,
-    model: t.Union["PathType", "tf_ext.KerasModel", "tf_ext.Module"],
```

Comment:
`save_model` should not accept pathtype, if we want to support import from filesystem then add another function like `copy_model` etc. later.